Extensions/show_more.js

```diff
@@ -869,6 +839,20 @@ XKit.extensions.show_more = new Object({
 
 	},
 
+	intercept_classic_menu_ask: function(event) {
+		event.preventDefault();
+
+		XKit.tools.add_function(function(){
+			var args = JSON.parse(add_tag);
+			var config = jQuery("<div data-tumblelog-name='" + args.recipient + "' />");
+			config.data("anonymous_ask", args.anonymous_asks);
```

Comment:
Is this config item supposed to add an "Ask as anonymous" control? My recipient blog definitely supports anonymous asks but I am not seeing such a control:

![anon](https://cloud.githubusercontent.com/assets/13437011/9201273/b751e3b2-4011-11e5-8bd5-5eb261ddef14.png)